src/witan/app/email.clj

```diff
@@ -0,0 +1,19 @@
+(ns witan.app.email
+  (:require [amazonica.aws.simpleemail :as ses]
+            [clojure.java.io :as io]
+            [clostache.parser :as parser]))
+
+(defn send-password-reset!
+  [username token]
+  (let [url (str "witan-alpha.mastodonc.com/#/password-reset/" token "/" username)
+        ctx {:link_to_reset_password url}
+        render #(-> (str "email-templates/reset-your-password" %)
+                    (io/resource)
+                    (slurp)
+                    (parser/render ctx))]
+    (ses/send-email {:endpoint "eu-west-1"}
```

Comment:
It works with my default witan AWS creds but yeah, since we changed to instance roles...I'm not sure, maybe something terraform can do? Can we still get this working on the current stack with roles?